client-internal/src/main/java/org/terracotta/angela/client/support/junit/AngelaRule.java

```diff
@@ -68,6 +68,12 @@ public AngelaRule(ConfigurationContext configuration, boolean autoStart, boolean
     this.autoActivate = autoActivate;
   }
 
+  public ConfigurationContext configure(ConfigurationContext configuration) {
+    ConfigurationContext old = this.configuration;
+    this.configuration = configuration;
+    return old;
+  }
```

Comment:
ClusterDefinition is class based.  We want to be able to redefine the config on a per method basis in case we want one method in a class run slightly different (Inline on/off)

Reply:
would it be better then to remove it from the ctor and instead have this method be called: `withConfiguration()` and return this ? or have a builder like the Junit Timeout rule ?

Reply:
I like the default with override personally.